src/server/models/pushSubscriptionModel.ts

```diff
@@ -10,48 +12,53 @@ interface PushSubscriptionInfo {
 }
 
 class PushSubscriptionModel {
-  private pushSubscriptions: {[id: string]: PushSubscriptionInfo[]};
+  private firestore: Firestore;
 
   constructor() {
-    this.pushSubscriptions = {};
+    this.firestore = new Firestore();
+  }
+
+  _getSubscriptionCollection(login: string) {
```

Comment:
Could you add a comment somewhere explaining the structure of the db (including the fact that it's keyed by base64 of endpoint etc.)